db/migrate/20160721211806_remove_unicode_table.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUnicodeTable < ActiveRecord::Migration
```

Comment:
You'll want to remove this file - it's causing the tests to fail, and will cause issues for other members' migrations.